swap_meet/vendor.py

```diff
@@ -1,2 +1,64 @@
+from .item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+
+        return False
+
+    def get_by_category(self, category):
+        result = [item for item in self.inventory if item.category is category]
+        return result
+
+    def get_best_by_category(self, category):
+        all_by_category = self.get_by_category(category)
+
+        if not all_by_category:
+            return None
+
+        max_rating = max(item.condition for item in all_by_category)
+        best_by_category = [
+            item for item in all_by_category if item.condition == max_rating
+        ]
+        # Assumed that in event of tie, return first item with highest rating
+        return best_by_category[0]
```

Comment:
Great use of `max` here! We could simplify this a little bit by changing how we use `max` by calling it on the with `all_by_category` list and passing a lambda function key which looks at the condition of each item for the comparison: 
```suggestion
        return max(all_by_category, key=lambda item: item.condition)
```